sysl2/sysl/eval/ValueUtil.go

```diff
@@ -236,6 +239,8 @@ func endpointToValue(e *sysl.Endpoint) *sysl.Value {
 	AddItemToValueMap(m, "attrs", attrsToValueMap(e.Attrs))
 	AddItemToValueMap(m, "is_rest", MakeValueBool(e.RestParams != nil))
 	AddItemToValueMap(m, "is_pubsub", MakeValueBool(e.IsPubsub))
+	retTypes := MakeValueMap()
+	retValues := []string{}
```

Comment:
Sorry, I missed this earlier. Please use nil slice declaration instead of empty slice.
`var retValues []string`

Reply:
ok